bindings/csharp/Regorus.Tests/RegorusTests.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Regorus.Tests;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Text.Json.Nodes;
+
+[TestClass]
+public class RegorusTests
+{
+    [TestMethod]
+    public void Basic_evaluation_succeeds()

```

Comment:
Interesting naming convention. Is this the standard for naming tests?

Reply:
I haven't really seen any standards around naming for C# tests. This is the convention I tend to follow + recommend - some examples from Bicep:

* https://github.com/Azure/bicep/blob/0b36c48266652fdae833fd171ff220dd9671d594/src/Bicep.Core.IntegrationTests/LambdaTests.cs#L87
* https://github.com/Azure/bicep/blob/0b36c48266652fdae833fd171ff220dd9671d594/src/Bicep.Core.IntegrationTests/LambdaTests.cs#L56
* https://github.com/Azure/bicep/blob/0b36c48266652fdae833fd171ff220dd9671d594/src/Bicep.Core.IntegrationTests/LambdaTests.cs#L227